.github/workflows/apply-precommit.yml

```
@@ -0,0 +1,32 @@
+name: Check and Fix Whitespace
```

I guess this should be:
```suggestion
name: Check Whitespace on Pull Request
```

?